api/v1alpha3/azurecluster_conversion.go

```diff
@@ -223,14 +237,30 @@ func Convert_v1alpha3_SecurityGroup_To_v1alpha4_SecurityGroup(in *SecurityGroup,
 	return nil
 }
 
-// Convert_v1alpha3_IngressRule_To_v1alpha4_IngressRule
-func Convert_v1alpha3_IngressRule_To_v1alpha4_IngressRule(in *IngressRule, out *infrav1alpha4.IngressRule, s apiconversion.Scope) error {
-	return autoConvert_v1alpha3_IngressRule_To_v1alpha4_IngressRule(in, out, s)
+// Convert_v1alpha3_IngressRule_To_v1alpha4_SecurityRule
+func Convert_v1alpha3_IngressRule_To_v1alpha4_SecurityRule(in *IngressRule, out *infrav1alpha4.SecurityRule, _ apiconversion.Scope) error { //nolint
+	out.Name = in.Name
+	out.Description = in.Description
+	out.Protocol = v1alpha4.SecurityGroupProtocol(in.Protocol)
+	out.Priority = in.Priority
+	out.SourcePorts = in.SourcePorts
+	out.DestinationPorts = in.DestinationPorts
+	out.Source = in.Source
+	out.Destination = in.Destination
```

Comment:
Just to confirm, `out.Direction = Inbound` is not needed here because it's set in defaults?

Reply:
correct

Reply:
added it anyways, doesn't hurt to make it explicit